common/cards/single-use/treasure-hunter.ts

```diff
@@ -0,0 +1,44 @@
+import {ITEM_CARDS} from '..'
+import {CardPosModel} from '../../models/card-pos-model'
+import {GameModel} from '../../models/game-model'
+import SingleUseCard from '../base/single-use-card'
+
+class TreasureHunterSingleUseCard extends SingleUseCard {
+	constructor() {
+		super({
+			id: 'treasure_hunter',
+			numericId: 152,
+			name: 'Treasure Hunter',
+			rarity: 'rare',
+			description: 'Shuffle 3 "Snow Block" coins into your deck',
```

Comment:
more naming inconsistency here for the coin card - I also see this as wayy too op for stall

Reply:
it's true that the icon used is a snow block however for us that just represents any type